language/go/generate.go

```diff
@@ -459,9 +468,24 @@ func (g *generator) generateLib(pkg *goPackage, embed string) *rule.Rule {
 	return goLibrary
 }
 
-func (g *generator) generateBin(pkg *goPackage, library string) *rule.Rule {
-	name := pathtools.RelBaseName(pkg.rel, getGoConfig(g.c).prefix, g.c.RepoRoot)
-	goBinary := rule.NewRule("go_binary", name)
+func (g *generator) generateAlias(pkg *goPackage, libName string) *rule.Rule {
+	if pkg.isCommand() || libName == "" {
+		return nil
+	}
+	c := getGoConfig(g.c)
```

Comment:
Nit: use the abbreviation `gc` for `*goConfig` to distinguish it from `*config.Config`.

Reply:
Done.